src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -113,8 +117,7 @@ static void DetectCiphersuiteConfiguration()
 
 #else
 
-    // The Fedora, RHEL, and CentOS builds replace the normal defaults (with a configuration model).
-    // Consider their non-portable builds to always have specified ciphersuites in config.
+    /// Produce g_config_specified_ciphersuites = 1 on OpenSSL 1.0.x.
```

Comment:
```suggestion
    // OpenSSL 1.0 does not support CipherSuites so there is no way for caller to override default
```

Just so it matches the other place where we positively assert it.